.github/workflows/release.yml

```diff
@@ -219,6 +219,9 @@ jobs:
           $Env:BAZEL_VC = "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC"
           set PreferredToolArchitecture=x64
 
+          # The default is "/arch:AVX", which we don't want.
+          $Env:CC_OPT_FLAGS = "/O2"
```

Comment:
Before adding this change, I did a test release that _didn't_ pass: https://github.com/larq/compute-engine/actions/runs/943850235

But I don't know if that's because this change actually made a difference that makes the builds pass, or if I'm just observing high build-time noise (which anecdotally seems to be the case).

Reply:
I guess this _could_ affect performance of the LCE interpreter on Windows, but:
a) We don't have optimised binary kernels on Windows anyway.
b) AFAIK even without this flag AVX intrinsics will insert AVX instructions, so TFL kernels with optimised Windows kernels should still be okay.

Reply:
Yeah I think this is a very reasonable thing to do.